js/bibox.js

```diff
@@ -1269,25 +1276,104 @@ module.exports = class bibox extends Exchange {
+        //         "order_side": 2,             // Trading side 1-Buy, 2-Sell
+        //         "order_type": 2,             // 2-limit order
+        //         "price": "0.00900000",       // order price
+        //         "amount": "1.00000000",      // order amount
+        //         "money": "0.00900000",       // currency amount (price * amount)
+        //         "deal_amount": "0.00000000", // deal amount
+        //         "deal_percent": "0.00%",     // deal percentage
+        //         "unexecuted": "0.00000000",  // unexecuted amount
+        //         "status": 3                  // Status,-1-fail, 0,1-to be dealt, 2-dealt partly, 3-dealt totally, 4- cancelled partly, 5-cancelled totally, 6-to be cancelled
+        //     }
+        //
+        let marketId = this.safeString (order, 'm');
+        if (marketId === undefined) {
+            const baseId = this.safeString (order, 'coin_symbol');
+            const quoteId = this.safeString (order, 'currency_symbol');
+            if ((baseId !== undefined) && (quoteId !== undefined)) {
+                marketId = baseId + '_' + quoteId;
+            }
         }
         market = this.safeMarket (marketId, market);
-        const rawType = this.safeString (order, 'order_type');
-        const type = (rawType === '1') ? 'market' : 'limit';
-        const timestamp = this.safeInteger (order, 'createdAt');
-        const price = this.safeString (order, 'price');
+        let type = this.safeString2 (order, 'T', 'order_type');
+        if (type === '1') {
+            type = 'limit';
+        } else if (type === '2') {
+            type = 'market';
+        }
+        const timestamp = this.safeInteger2 (order, 'C', 'createdAt');
+        const price = this.safeString2 (order, 'P', 'price');
         const average = this.safeString (order, 'deal_price');
         const filled = this.safeString (order, 'deal_amount');
-        const amount = this.safeString (order, 'amount');
-        const cost = this.safeString2 (order, 'deal_money', 'money');
-        const rawSide = this.safeString (order, 'order_side');
-        const side = (rawSide === '1') ? 'buy' : 'sell';
-        const status = this.parseOrderStatus (this.safeString (order, 'status'));
-        const id = this.safeString (order, 'id');
+        let amount = this.safeString2 (order, 'Q', 'amount');
+        amount = Precise.stringAbs (amount);
+        const cost = this.safeString2 (order, 'money', 'deal_money');
+        let side = this.safeString2 (order, 's', 'order_side');
+        if (side === '1') {
+            side = 'buy';
+        } else if (side === '2') {
+            side = 'sell';
+        }
+        const status = this.parseOrderStatus (this.safeString (order, 'S', 'status'));
+        const id = this.safeString2 (order, 'i', 'id');
+        const clientOrderId = this.safeString (order, 'I');
+        const timeInForce = this.safeString (order, 't');
+        const postOnly = this.safeValue (order, 'o');
```

Comment:
I think we need to adjust `fee` parsing as well 
![image](https://user-images.githubusercontent.com/43336371/196222023-9df90eba-5df5-42b4-942d-0175125700ed.png)